build_tools/cmake/run_tf_tests.sh

```diff
@@ -13,6 +13,7 @@ cd "${ROOT_DIR}"
 
 BUILD_DIR="$1"
 IREE_VULKAN_DISABLE="${IREE_VULKAN_DISABLE:-0}"
+IREE_LLVM_CPU_DISABLE="${IREE_VULKAN_DISABLE:-0}"
```

Comment:
this seems wrong?

```suggestion
IREE_LLVM_CPU_DISABLE="${IREE_LLVM_CPU_DISABLE:-0}"
```

Reply:
It does indeed seem that way, but I'm actually galaxy brain on this one :stuck_out_tongue: